recipe/0001-disable-protobuf.patch

```diff
@@ -0,0 +1,856 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 66931b0..e44a113 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -9,7 +9,6 @@ if(MSVC)
+   endif()
+ endif()
+ 
+-option(ENABLE_PROTOBUF "Enable Protobuf" OFF)
```

Comment:
This patch seems fair, but is it necessary? Seems like we could just rely on the cmake option to disable protobuf and not worry about eliminating the code.

Reply:
I mechanically applied dmlc/treelite#193, since this pull request was tested and merged into the main branch.

Reply:
Seems like a fair argument to me!